database_admin/update.go

```diff
@@ -99,14 +99,19 @@ func dbConn() (database.Driver, *sql.DB) {
 func UpdateDB(migrationFilesURL string) {
 	utils.ConfigureLogging()
 	migrationEnabled := utils.GetBoolEnvOrDefault("ENABLE_MIGRATION", false)
+
+	conn, db := dbConn()
+	getAdvisoryLock(db)
+	defer releaseAdvisoryLock(db)
```

Comment:
@psegedy
- Could you briefly sum up, why it was bad before and what it fixes after change?
- If it sleeps on select, isn't it dangerous to hold AdvisoryLock?

Reply:
- it blocked deployement when you had ENABLE_MIGRATION=false but you didn't need to make any migrations
- no, it should block the new deployment

Reply:
@michalslomczynski do you know a scenario when it could be a problem that pgAdvisoryLock is not released?

Reply:
Not on top of my head at the moment